### PR TITLE
Adding Builder's Journey to ./guides (as README.md)

### PR DESCRIPTION
**Pull request by albrich-me**
**Issue #:**

**Description of changes:**  Added Builder's Journey as README.md in /guides.  Added link to /guides called Getting Started Guides in ./README.md 

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*

**Checklist:**

1. Have you successfully run `npm run build:release` locally?

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: guides/README.md [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+### 1.1 What is Amazon Chime SDK (and what isn&#39;t it)?
+
+The Amazon Chime SDK provides a rich set of APIs and example code that enables customers to build modern communication applications that easily interface to other AWS services, including Connect, Pinpoint, Lex, Polly, Transcribe, and others. The Amazon Chime SDK has three key pillars:
+
+Meetings, including audio/video via browser(WebRTC), iOS, or Android
+
+Messaging
+
+PSTN Audio, including SIP Trunking
+
```
[the remark this answers is not in the document]
Comment (albrich-me) [reply guides/README.md:38]: Bullet Points added.

File: guides/README.md [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+[Amazon Chime SDK for Android](https://docs.aws.amazon.com/chime/latest/dg/sdk-for-android.html) - core code to support building Android-based collaboration apps
+
+[Amazon Chime SDK Messaging](https://aws.amazon.com/chime/chime-sdk/features/#Messaging) – The Messaging feature of the SDK allows you to create persistent messaging applications that can exist either within or outside of a meeting. Click [here](https://aws.amazon.com/chime/chime-sdk/features/#Messaging) for more details.
+
+[Amazon Chime SDK for PSTN Audio](https://aws.amazon.com/chime/chime-sdk/features/#Public_Switched_Telephone_Network_.28PSTN.29_Audio) – The PSTN feature of the SDK supports a number of functions but the ones we will be focused on will be those that relate to meetings including Inbound Calling and Outbound Calling. Click [here](https://aws.amazon.com/chime/chime-sdk/features/#Public_Switched_Telephone_Network_.28PSTN.29_Audio) for more details.
+
+For more information on the Amazon Chime SDK offering see:
+
+Amazon Chime SDK Website
```
[the remark this answers is not in the document]
Comment (albrich-me) [reply guides/README.md:65]: Hyperlink Added

File: guides/README.md [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+
+For each AWS account there are hard limits that cannot be adjusted and soft limits which can be adjusted:
+
+**Hard limits:**
+
+250 users per session
+
+100 video sources on simultaneously
+
+25 tiles displayed simultaneously
```
[the remark this answers is not in the document]
Comment (albrich-me) [reply guides/README.md]: I updated the document to be more consistent and pushed it to my fork of the repo.  Let me know if I need to do anything else.

**Review by michhyun1 — APPROVED** [review]

**Comment by michhyun1** [comment]
I think you need to run:` npm run build:release`

**Comment by michhyun1** [comment]
Closing this PR in favor of https://github.com/aws/amazon-chime-sdk-js/pull/2417

**Pull request closed without merging**